pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -229,6 +229,10 @@ type GlobalRoleBinding struct {
 	// +optional
 	GroupPrincipalName string `json:"groupPrincipalName,omitempty" norman:"noupdate,type=reference[principal]"`
 
+	// UserPrincipalName is the name of the user principal subject added to the cluster. Immutable.
```

Comment:
Nit: It applies to more than just the cluster, so maybe use the same verbiage as above
```suggestion
	// UserPrincipalName is the name of the user principal subject to be bound. Immutable.
```